app/packages/looker/src/worker.ts

```diff
@@ -169,7 +186,8 @@ const getSendChunk = (uuid: string) => ({
     let buffers: ArrayBuffer[] = [];
 
     value.frames.forEach((frame) => {
-      buffers = [...buffers, ...processMasks(frame)];
+      delete frame._sample_id;
```

Comment:
Can this be easily generalized? The idea is to exclude all private fields except ones that we're manually opting into showing (for both sample- and frame-level fields).

For samples, `_rand` is currently the only private field we're not showing, and `_sample_id` is the only frame-level field, but presumably more will come in the future.

Reply:
I'll put it on the backend, but really this goes back to a data model overhaul. The App is a client, so it shouldn't have to modify representation ever.

Reply:
Ok, this line was unnecessary. Pardon the delay.

I think `filterSample` in [`index.ts`](https://github.com/voxel51/fiftyone/pull/1208/files#diff-d526c437fbb7dd06767fb2fb657232d0354eccffee28c5ef4536871d23d0e0b6),  which handles `_sample_id`,  is fairly robust and general. It leverages the fields maps (frame and sample). So I'm thinking it's fine to leave this in `Looker` until some more server work is done.

Reply:
👍